### PR TITLE
Add buttons

### PR DESCRIPTION
**Pull request by justincamacho**

### CODE REVIEW

**Review by suwdo — APPROVED** [review]
Just needs the package.json/lock ver number increment

**Review by suwdo — APPROVED** [review]
😼

**Review by aapowers — APPROVED** [review]
a giles button? haha

**Review by aapowers — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ucirvine/eeeplus-theme",
-  "version": "0.5.2",
+  "version": "1.0.0",
```
Comment (cmelzer) [inline package.json:3]: WooHoo, version 1!

**Review by cmelzer — APPROVED** [review]

File: docs/index.html [hunk 2]
```diff
@@ -229,6 +217,54 @@ <h2><a href="#buttons">#</a>Buttons</h2>
...
+            <button class="btn btn-primary long-cta-btn-sm">
+                small 2            </button>
+        </div>
+        
+
+        <div class="pb-2 pr-1">
+            <button class="btn disabled btn-primary long-cta-btn-sm">
+                small 2            </button>
+        </div>
+
```
Comment (dperezpe) [inline docs/index.html:257]: are these all meant to show up twice? small shows up a few times

Comment (dperezpe) [reply docs/index.html:257]: oh i see they have other classes that are different

**Review by dperezpe — APPROVED** [review]

**Pull request merged**